bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/i18n/tapocontrol.properties

```diff
@@ -77,3 +81,24 @@ channel-type.tapocontrol.outputChannel.label = Output Switch
 channel-type.tapocontrol.outputChannel.description = Switches the power state on/off
 channel-type.tapocontrol.overheated.label = Device Overheated
 channel-type.tapocontrol.overheated.description = ON if device is overheated
+channel-type.tapocontrol.todayEnergyUsageChannel.label = Today Usage
+channel-type.tapocontrol.todayEnergyUsageChannel.description = Today energy usage
+channel-type.tapocontrol.todayRuntimeChannel.label = Today Runtime
+channel-type.tapocontrol.todayRuntimeChannel.description = Today runtime (On-Time)
+
+# thing types
+
+thing-type.tapocontrol.Test_Device.label = Tapo Universal TestDevice
+thing-type.tapocontrol.Test_Device.description = For testing pruposes! Response is written down as info in openhab-log
+
+# channel group types
+
+channel-group-type.tapocontrol.commandDebug.label = Device Communication Debug
+channel-group-type.tapocontrol.commandDebug.description = Device resoponses and command debugging
+
+# channel types
+
+channel-type.tapocontrol.deviceCommand.label = Device Command
+channel-type.tapocontrol.deviceCommand.description = command send to device. use: 'command':'value'
+channel-type.tapocontrol.deviceResponse.label = Device Response
+channel-type.tapocontrol.deviceResponse.description = DeviceResponse
```

Comment:
Why all these entries are now added while I do not see corresponding stuff added in your changes ?
Was it previously missing ?

Reply:
This were generated from testdevice in \src\test folder. I removed it in .properties now